project/plugins.sbt

```diff
@@ -1,13 +1,18 @@
-resolvers += Resolver.url("hmrc-sbt-plugin-releases", url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
-
-resolvers += "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
+resolvers ++= Seq(
+  Resolver.url("hmrc-sbt-plugin-releases",
+    url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))
+  (Resolver.ivyStylePatterns),
+  "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
+)
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "1.7.0")
```

Comment:
Catalogue mentions 1.8.0 is latest version of sbt-auto-build plugin (https://catalogue.tax.service.gov.uk/service/agent-fi-relationship)

Reply:
:+1: , what about the other items that are minorly behind in this catalogue? Shall I upgrade those as well?